packages/SwingSet/src/liveslots/liveslots.js

```diff
@@ -1181,6 +1181,27 @@ function build(
     ...collectionManager.testHooks,
   });
 
+  function setVatOption(option, value) {
+    switch (option) {
+      case 'virtualObjectCacheSize': {
+        if (typeof value === 'number') {
+          vom.setCacheSize(value);
```

Comment:
I'd definitely `Nat()` this

Reply:
Yup.  As I've been cleaning these up I've decided I like `isNat()` much more than `Nat()`